docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -113,7 +116,18 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
         (warehouse, location),
         (warehouse, size),
         (warehouse))`.
- 
+    
+    `GROUP BY warehouse, GROUPING SETS((env), GROUPING SETS((product), ()), GROUPING SETS((location, size), (location), (size), ()))`
+    and `GROUP BY warehouse, GROUPING SETS((env), ROLLUP(product), CUBE(location, size))` is equivalent to 
+    `GROUP BY  GROUPING SETS(
```

Comment:
It's better to give a simple example like `GROUP BY GROUPING SETS(GROUPING SETS(warehouse, product), GROUPING SETS(location, size))`

Reply:
Done